pom.xml

```diff
@@ -2099,6 +2099,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.yaml</groupId>
+                <artifactId>snakeyaml</artifactId>
```

Comment:
the PR is missing the license stuff for this new dependency

Reply:
iiuc snakeyaml is already used by WildFly. Could you then create a WFLY issue to remove `snakeyaml` from WildFly when we will eventually upgrade a version of WildFly Core that contains this feature